source/vir/src/sst.rs

```diff
@@ -64,6 +64,7 @@ pub enum ExpX {
     Bind(Bnd, Exp),
     // only used internally by the interpreter; should never be seen outside it
     Interp(InterpExp),
+    InternalFun(InternalFun, Typs, Exps),
```

Comment:
I like the way `InternalFun` distinguishes between real calls and internal calls.  But rather than adding another case to ExpX, I'd rather share code between `InternalFun` and `Call` by using a `CallTarget`-like enum for the target of `ExpX::Call`, something like `Call(CallFun, Typs, Exps)`, with the `InternalFun` cases going into `CallFun`.